### PR TITLE
add sanity test boilerplate for groups

### PR DESCRIPTION
**Pull request by ryanfkeepers**
#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [x] :robot: Supportability/Tests

#### Issue(s)

* #3989 

#### Test Plan

- [x] :green_heart: E2E

### CODE REVIEW

File: .github/workflows/sanity-test.yaml [hunk 1]
```diff
@@ -318,6 +321,53 @@ jobs:
...
           log-dir: ${{ env.CORSO_LOG_DIR }}
+          with-export: true
+
+##########################################################################################################################################
+
+# Groups and Teams
+
+      # generate new entries for test
+      - name: Groups - Create new data
+        if: never() # TODO: enable when ready
```
Comment (meain) [inline .github/workflows/sanity-test.yaml:332]: I don't think `never()` is available. I think we can however just use `false` though.

File: .github/actions/backup-restore-test/action.yml [hunk 2]
```diff
@@ -52,6 +56,7 @@ runs:
             tee $GITHUB_OUTPUT
 
     - name: Restore ${{ inputs.service }} ${{ inputs.kind }}
+      if: inputs.restore != ""
```
Comment (meain) [inline .github/actions/backup-restore-test/action.yml:59]: I'm assuming you meant `restore-args`.

File: .github/actions/backup-restore-test/action.yml [hunk 3]
```diff
@@ -86,10 +92,10 @@ runs:
         ./sanity-test
 
     - name: Export ${{ inputs.service }} ${{ inputs.kind }}
+      if: ${{ inputs.with-export == true }}
```
Comment (meain) [inline .github/actions/backup-restore-test/action.yml:95]: Nit: do we need the `== true` check?

**Review by meain — APPROVED** [review]
Need a few changes,  but approving to unblock.

File: .github/actions/backup-restore-test/action.yml (hunk 2 above)
Comment (ryanfkeepers) [reply .github/actions/backup-restore-test/action.yml:59]: whoops!  Thank you, yes.

File: .github/workflows/sanity-test.yaml (hunk 1 above)
Comment (ryanfkeepers) [reply .github/workflows/sanity-test.yaml:332]: so it seems.  Good catch.

File: .github/actions/backup-restore-test/action.yml (hunk 3 above)
Comment (ryanfkeepers) [reply .github/actions/backup-restore-test/action.yml:95]: probably not.  But it's not really a bug.  Just excess.

**Pull request merged**